docs-styleguide.md

```diff
@@ -31,6 +31,7 @@ This is the documentation styleguide for our natural language descriptions used
 
 * End phrases with a period. (Like that one).
 * Capitalize the first letter of a phrase or sentence.
+* Command & subcommand taglines, (and option descriptions when possible) should use the imperative mood. For example: "Clear a field.", "Interact with the DHT."
```

Comment:
No; it's not a tense. It is a mood. Tenses pertain to place in time; mood pertains to modality.